packages/connect-core/src/entities/Organization.ts

```diff
@@ -125,14 +154,83 @@ export default class Organization {
   }
 
   ///////// INTENTS ///////////
-  appIntent(
-    appAddress: Address,
-    functionName: string,
-    functionArgs: any[]
-  ): TransactionIntent {
-    return new TransactionIntent(
-      { contractAddress: appAddress, functionName, functionArgs },
-      this,
+
+  /**
+   * Calculate the transaction path for a transaction to an external `destination`
+   * (not the currently running app) that invokes a method matching the
+   * `methodAbiFragment` with `params`.
+   *
+   * @param  {string} destination Address of the external contract
+   * @param  {object} methodAbiFragment ABI fragment of method to invoke
+   * @param  {Array<*>} params
+   * @param  {Object} options
+   * @return {Promise<ForwardingPath>} An object that represents the forwarding path corresponding to an action.
+   *   If the destination is a non-installed contract, always results in a
+   *   single step transaction.
+   */
+  async intent(
+    destination: AppOrAddress,
+    methodAbiFragment: ethersUtils.FunctionFragment,
+    params: any[],
+    options?: PathOptions
+  ): Promise<ForwardingPath | undefined> {
+    const sender = options?.actAs || this.connection.actAs
+    if (!sender) {
+      throw new Error(
+        `No sender address specified. Use 'actAs' option or set one as default on your organization connection.`
+      )
+    }
+
+    const installedApps = await this.apps()
+
+    return organizationIntent(
+      sender,
+      destination,
+      methodAbiFragment,
+      params,
+      installedApps,
+      this.connection.ethersProvider
+    )
+  }
+
+  //////// DESCRIPTIONS /////////
+
+  // Return a description of the forwarding path encoded on the evm script
+  async describeScript(script: string): Promise<ForwardingPathDescription> {
```

Comment:
Provide access to the consumer for the `describeScript` helper in the context of the Connect connection.